CONTRIBUTING.md

```diff
@@ -10,6 +10,17 @@ Please note we have a code of conduct, please follow it in all your interactions
 2. Before contributing to an issue, make sure that no one is assigned or has taken that issue. If no one is and you would like to work on the issue, please comment on the issue to let others know that someone is working on the issue.
 3. Before creating a pull request, it is **important** that you make sure your branch and repository is up to date with this one. Some conflicts can be resolved, but many are hard to resolve. **Add the upstream** branch and always keep your branch up to date.
 
+- [Instructions](#instructions)
```

Comment:
Please add the title "Content" to it

Reply:
Already did it sorry, I didn't like how it looked :)